react/week2/todo/src/components/Todo.js

```diff
@@ -0,0 +1,74 @@
+import React, { useState } from "react";
+
+function Todo(props) {
+  const [state, setDone] = useState([...props.items]);
+
+  const addTodo = () => {
+    // Add random item
+    const addRandomItem = (arr) => {
+      const newId = arr.length
+        ? Math.max(...arr.map((item) => item.id)) + 1
+        : 1;
+      const addRandomArr = arr.concat({
```

Comment:
The naming of this variable is a bit confusing. Verbs like "add" are used in function names, but this is the value of the `concat` array method. I would name this something like `newTodos`.